comm/README.md

```diff
@@ -20,57 +20,111 @@ Clone (or download and unpack) the master branch of the source.
 
 ### Building with `sbt`
 
-The only up-front build-time requirement is [sbt](http://www.scala-sbt.org/download.html), which should be installed
-according to your platform. It, in turn, will download and include all dependencies for the system.
+The only up-front build-time requirements are the Java Development Toolkit (we've been using [OpenJDK version
+1.8.0_151](http://openjdk.java.net/install/)) and [sbt](http://www.scala-sbt.org/download.html), both of which should be installed
+according to your platform.
 
-To build once `sbt` is installed, issue:
+To build, simply issue from a terminal:
 
 ```
 sbt assembly
 ```
 
-This will download dependencies, if required, compile all the code, construct a single, large jar file, and run all the
-tests.
+This will download dependencies, if required, compile all the code, construct a single, large jar file, and run all the tests.
 
 ### Building via Docker
 
-If you have [docker](https://www.docker.com/) installed, you can build a docker image. Under the covers, the docker
-build process is the `sbt` build process, exactly as described above. The command
+The only Docker combination that has been extensively tested is
+
+```
+Client:
+ Version:      17.09.1-ce
```

Comment:
Is this the Docker version or the Ubuntu one?

Reply:
That's the version of the `docker` command; the one below is the version of the daemon that it talks to (by default, in this case).